Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM openjdk
+FROM openjdk:8-jdk-alpine
```

Comment:
👍, though once we get to the point we're not actively developing the `Dockerfile` itself we should probably switch to a specific (hash-equivalent) version.